blog/2015/12/three-flavors-of-instructor-training.html

```diff
@@ -0,0 +1,124 @@
+  it's a nice tool,
+  but wasn't noticeably better for our purposes than the shared WordPress blog we used to use.
+  (Both are much easier to manage than the GitHub-based blog we used in the winter, though.)
+  We used BlueJeans for the hour-long meetings,
+  which worked as well as web conferencing ever does.
+</p>
+<p>
+  For the two-day version of the pilot we had people apply in groups.
+  One reason was so that each site would have enough learners to do
+  the video recorded teaching exercise that has become such an important part of the in-person course.
+  Another was so that there'd be critical mass to run workshops after the training:
+  participants have said in the past that this is one of the big challenges.
+  Nine groups took part in total:
+</p>
+<table class="table table-striped">
+  <tr>
+    <th>Team</th>
+    <th>Location</th>
+    <th>Size</th>
+  </tr>
+  <tr>
+    <td>Team 1</td>
+    <td>Townsville</td>
+    <td>4</td>
+  </tr>
+  <tr>
+    <td>Team 2</td>
+    <td>Thessaloniki</td>
+    <td>4</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Potsdam</td>
+    <td>5</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Paris</td>
+    <td>7</td>
+  </tr>
+  <tr>
+    <td>Team 3</td>
+    <td>Toronto</td>
+    <td>14</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Curitiba</td>
+    <td>3</td>
+  </tr>
+  <tr>
+    <td>Team 4</td>
+    <td>Middleton</td>
+    <td>7</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Arlington</td>
+    <td>10</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Vancouver</td>
+    <td>5</td>
+  </tr>
+</table>
+<p>
+  Each team used one Google Hangout;
+  there were a few glitches,
+  but it was mostly reliable.
+  (A DDoS attack on UK academic networks during the training didn't help matters,
+  but Team 2 managed nonetheless.)
+</p>
+<p>
+  Overall,
+  we think that the two-day version worked as well as the multi-week version,
+  and was definitely less effort for both instructors and learners.
+  A few of the learners in the two-day version may have been less enthusiastic than usual
+  (we think they may have been roped in by friends to get numbers up for applications),
+  but on the other hand,
+  all but a couple of the people who started the two-day version finished it.
+  There wasn't much inter-group discussion&mdash;sessions tended to be hub-and-spoke with the trainer as the hub&mdash;but
+  there seemed to be plenty of conversation within each group.
+  The biggest drawback of the two-day version was that instructors couldn't give the same quality of feedback on exercises
+  as in either the live version or the multi-week version:
+  there wasn't as much time to do so as in the latter,
+  and there wasn't the hands-on aspect of the former (particularly for the live coding exercise).
+  On the other hand,
+  the online live course made a lot more use of the Etherpad,
+  which meant there were good collaboratively-produced notes at the end.
+</p>
```

Comment:
Not sure if a note on Manchester F2F instructor training could go here but:

"Around the same time (23-24 November) we also run a face-to-face instructor training at the University of Manchester. This session was supported by ELIXIR UK and the places for trainees were offered to members of the ELIXIR project which meant that we had a very international group present. Whilst this maybe did not made a significant difference to the training itself (apart from an opportunity to discuss some cultural differences relevant to the teaching practices), it was great to see the international outreach and growth of Software and Data Carpentry. For newly trained instructors this is an incredibly positive factor and many attendees mentioned how much they enjoyed the fact that they could network and meet their counterparts from other countries and organisations. 

The Instructor Training in Manchester was run by Greg Wilson with help from Aleksandra Pawlik and Karin Lagesen. In particular for Karin who is about to complete her training as instructor trainer, it was possible to see how the session is run in face to face setting. 

Inevitably, having the whole group in one physical location means that there is a lot of discussion and interaction (yet, the trainer has to always help facilitate that, especially at the beginning). It also means that for the practical exercises the trainers can ensure that people swap the partners/groups which increases interaction and knowledge exchange."